notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+    "I0716 17:55:44.000355 313 model_repository_manager.cc:1212] successfully loaded 'fil' version 1\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Stopping Triton Server before proceeding further\n",
+    "os.killpg(os.getpgid(pro.pid), signal.SIGTERM)  # Send the signal to all the process groups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dynamic Batching\n",
+    "\n",
+    "Dynamic batching is a feature of Triton that allows inference requests to be combined by the server, so that a batch is created dynamically. This results in increased throughput. To perform dynamic batching in Triton you'd need to modify the following property in config.pbtxt file:\n",
+    "```\n",
+    "dynamic_batching {\n",
+    "  preferred_batch_size: [1024, 8192]\n",
+    "  max_queue_delay_microseconds: 100\n",
+    "}\n",
+    "```\n",
+    "The *preferred_batch_size* property indicates the batch sizes that the dynamic batcher should attempt to create. The *max_queue_delay_microseconds* property setting delays sending the batch, when a batch of a preferred size cannot be created from the available requests as long as no request is delayed longer than the configured max_queue_delay_microseconds value. If a new request arrives during this delay and allows the dynamic batcher to form a batch of a preferred batch size, then that batch is sent immediately for inferencing. If the delay expires the dynamic batcher sends the batch as is, even though it is not a preferred size."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model Analyzer\n",
+    "\n",
+    "[Triton Model Analyzer](https://github.com/triton-inference-server/model_analyzer) is a tool to profile and evaluate the best deployment configuration that maximizes inference performance of your model when deployed in Triton Inference Server. Using this tool, you can find the appropriate batch size, #TODO benefits of model analyzer. Model Analyzer installation steps are available here: [link](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/install.md)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install nvidia-pyindex"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install triton-model-analyzer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!export DCGM_VERSION=2.0.13\n",
+    "!wget -q https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/datacenter-gpu-manager_${DCGM_VERSION}_amd64.deb && \\\n",
```

Comment:
I thought DCGM was already in the Triton image? You're probably right that it's not, but I just want to make sure we're not installing this unnecessarily.